src/axel.h

```diff
@@ -132,4 +132,12 @@ void print_messages( axel_t *axel );
 
 double gettime();
 
+static inline int axel_nanosleep( struct timespec *delay_in )
```

Comment:
If using a pointer, it should be `const`.

Reply:
As mentioned previously, you can just pass the `struct` by value.

Reply:
yeah, ok. I will go for the struct by value